appmanifest/id-member/id-member-manual-tentative-v2.webmanifest

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "id member test",
+  "id": "testId",
```

Comment:
Can you make the in this one be url-equiv but not string-equiv? That will help make sure other user agents implement the equivalency correctly.

Reply:
good idea!